lib/lib-storage/src/Upload.spec.ts

```diff
@@ -26,6 +26,8 @@ const listPartsMock = jest.fn().mockResolvedValue({
     {
       PartNumber: 1,
       ETag: "mock-part-ETag",
+      // this was obtained by adding a console.log when calculating in actual code
+      ChecksumSHA256: "eu1MyFbmILcxZCObrKejv2ZFvYB9jcO8XX370ONoEHU=",
```

Comment:
Is this a security concern? I'm not familiar at all w/ Checksum. Curious to hear!